### PR TITLE
Turn-off autocomplete and correct for subreddit search

### PR DESCRIPTION
**Pull request by schwers**
Finding a subreddit by name is annoying, particularly with iOS's aggressive autocorrect

### CODE REVIEW

**Comment by schwers** [comment]
trying with autocomplete on in a bit, as its actually very good on android

File: src/views/components/CommunitySearchRow.jsx [hunk 1]
```diff
@@ -36,6 +36,9 @@ class CommunitySearchRow extends BaseComponent {
               id='location-input'
               placeholder='find a community'
               name='location'
+              autoCorrect='off'
+              autoCapitalize='off'
+              spellcheck='false'
```
Comment (ajacksified) [inline src/views/components/CommunitySearchRow.jsx]: `spellCheck`

**Comment by ajacksified** [comment]
:nail_care:

**Comment by schwers** [comment]
:haircut:

**Comment by ajacksified** [comment]
:fish:

**Pull request merged**